meinberlin/apps/contrib/assets/helpers.js

```diff
@@ -21,6 +21,7 @@ export const toLocaleDate = (
   locale = 'de-DE',
   formatStyle = { dateStyle: 'long' }
 ) => {
-  const date = new Date(isodate)
+  const normalizedDate = isodate.replace(/ /g, 'T')
+  const date = new Date(normalizedDate)
```

Comment:
nice! but can you add to the comment to explain why we add this, we will definitely forget :p